[2025-04-16T19:33:05.568+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: churn_pipeline_dag.ingest_csv manual__2025-04-16T19:32:59.766024+00:00 [queued]>
[2025-04-16T19:33:05.574+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: churn_pipeline_dag.ingest_csv manual__2025-04-16T19:32:59.766024+00:00 [queued]>
[2025-04-16T19:33:05.574+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-16T19:33:05.589+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): ingest_csv> on 2025-04-16 19:32:59.766024+00:00
[2025-04-16T19:33:05.599+0000] {standard_task_runner.py:57} INFO - Started process 58 to run task
[2025-04-16T19:33:05.603+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'churn_pipeline_dag', 'ingest_csv', 'manual__2025-04-16T19:32:59.766024+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/churn_pipeline_dag.py', '--cfg-path', '/tmp/tmp6qq6az0q']
[2025-04-16T19:33:05.607+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask ingest_csv
[2025-04-16T19:33:05.643+0000] {task_command.py:416} INFO - Running <TaskInstance: churn_pipeline_dag.ingest_csv manual__2025-04-16T19:32:59.766024+00:00 [running]> on host 333f9ee0262a
[2025-04-16T19:33:05.714+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='churn_pipeline_dag' AIRFLOW_CTX_TASK_ID='ingest_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-04-16T19:32:59.766024+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-16T19:32:59.766024+00:00'
[2025-04-16T19:33:05.850+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-16T19:33:05.856+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=churn_pipeline_dag, task_id=ingest_csv, execution_date=20250416T193259, start_date=20250416T193305, end_date=20250416T193305
[2025-04-16T19:33:05.907+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-16T19:33:05.918+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
